nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -521,21 +595,25 @@ define([
          * @param {DOMElement} node - the node that should be used for the left column
          */
         function buildFileTypePanel(node) {
+            let fileTypesDisplay = {};
+            let fileTypeMapping = {};
+            let uploaders = Config.get('uploaders');
+            for (const uploader of uploaders.dropdown_order) {
+                fileTypeMapping[uploader.id] = uploader.name;
+            }
+            for (const fileType of Object.keys(typesToFiles)) {
+                fileTypesDisplay[fileType] = {
+                    label: fileTypeMapping[fileType] || `Unknown type "${fileType}"`
+                };
+            }
```

Comment:
Here, we use the same configuration as the staging area to map from file type ids like `'fastq_reads'` to the label string `'FASTQ Reads'`. That's not exactly what's in the Figma mockups, so we might want to discuss if that's a problem.

Reply:
IIRC the design mockups were geared towards creating more separation between the FASTQ reads types, so that might be why the title is slightly different. either way agree that it's worth discussing to make sure we're on the right track